docs/language/crypto.mdx

```diff
@@ -173,9 +173,16 @@ The validation of the public key depends on the supported signature scheme:
 
 - `BLS_BLS_12_381`:
   The given key is correctly serialized following the compressed serialization in [IETF draft-irtf-cfrg-pairing-friendly-curves-08](https://www.ietf.org/archive/id/draft-irtf-cfrg-pairing-friendly-curves-08.html#name-point-serialization-procedu).
-  The coordinates represent valid prime field extension elemnents. The resulting point is on the curve, and is on the correct subgroup G_2. 
+  The coordinates represent valid prime field extension elements. The resulting point is on the curve, and is on the correct subgroup G_2. 
+  Note that the point at infinity is accepted and yields the identity public key. Such identity key can be useful when aggregating multiple keys.
 
-Since the validation happen only at the time of creation, public keys are immutable.
+<Callout type="info">
+
+🚧 Status: Accepting the BLS identity key is going to be available in the upcoming release of Cadence on Mainnet.
```

Comment:
maybe reference a Cadence version here? same for the other callout

Reply:
Good idea. We can update it later once we know when we know in what spork this becomes available.